SimDataFormats/GeneratorProducts/src/classes_def.xml

```diff
@@ -16,7 +16,8 @@
     <class name="HepMC::GenCrossSection" ClassVersion="10">
      <version ClassVersion="10" checksum="920043842"/>
     </class>
-	<class name="HepMC::WeightContainer" ClassVersion="14">
+	<class name="HepMC::WeightContainer" ClassVersion="15">
+  <version ClassVersion="15" checksum="376377869"/>
```

Comment:
Checksum change is from removal of lines below. They cause warning which fails genreflex generation:
Warning: Unused class rule: std::iterator<std::forward_iterator_tag,HepMC::GenParticle*>
Warning: Unused class rule: std::iterator<std::forward_iterator_tag,HepMC::GenVertex*>